templates/kubernetes/terraform/modules/kubernetes/mysql/main.tf

```diff
@@ -0,0 +1,24 @@
+provider "mysql" {
```

Comment:
I thought the plan was still to use the `db-ops` image.. using the db providers here won't work because you won't have access to the databases.

Reply:
yah, that's the situation. We have to go through db-ops inside cluster